test/libsolidity/semanticTests/viaYul/array_storage_index_boundary_test.sol

```diff
@@ -19,8 +19,8 @@ contract C {
 // test_boundary_check(uint256,uint256): 10, 10 -> FAILURE, hex"4e487b71", 0x32
 // test_boundary_check(uint256,uint256): 256, 256 -> FAILURE, hex"4e487b71", 0x32
 // gas irOptimized: 668136
-// gas legacy: 648515
-// gas legacyOptimized: 628739
+// gas legacy: 648530
+// gas legacyOptimized: 628754
```

Comment:
Not sure why only these few increase a tiny bit, but nothing else is unaffected.